bin/console

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env ruby
```

Comment:
@gregg-platogo  @tmarkanov There is already something similar via `bundle exec rake console` which automatically instantiates a client.

Reply:
Right we missed that! BTW it would be cool if you added CHANGELOG to the releases 😉

Reply:
Done! https://github.com/kyledecot/app_store_connect/blob/main/CHANGELOG.md

Reply:
Awesome! Thanks!